manifests/p/ProaData/B2BClient/1.1.0/ProaData.B2BClient.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: ProaData.B2BClient
+PackageVersion: 1.1.0

```

Comment:
```suggestion
PackageVersion: 1.1.0.0
```

Reply:
It is necessary that we use 1.1.0.0 instead of 1.1.0?